src/Fixer/Whitespace/StatementIndentationFixer.php

```diff
@@ -326,6 +326,12 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                         continue;
                     }
 
+                    if ($endToken->isGivenKind(CT::T_ARRAY_SQUARE_BRACE_OPEN)) {
+                        $endIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_ARRAY_SQUARE_BRACE, $endIndex);
+
+                        continue;
+                    }
```

Comment:
It is difficult to understand the internals of StatementIndentationFixer. So I'm not sure if the fix is correct at this place.
But tests seem to be green.

Reply:
It is, great work finding the place 👍🏼